### PR TITLE
[New integration part 2] Prisma Cloud Compute - add playbooks, layouts, fields, classifier and other minor fixes

### PR DESCRIPTION
**Pull request by kostakhv**
## Status
- [x] Ready

## Description
- Add playbooks, layouts, fields, classifier
- Minor fixes to scripts
- Added headers data to integration, so that markdown table headers would be arranged properly
- Parse received data for any additional headers that might have been added
- Added UT to test the get_headers func

### CODE REVIEW

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+id: 143db226-930b-4b0a-8e37-3cf4805f7637
```
Comment (anara123) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Audit_Alert.yml]:
```suggestion
id: Prisma Cloud Compute - Audit Alert
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Cloud_Discovery_Alert.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+id: fbe45703-28f7-4906-8669-8a2c5e629df2
```
Comment (anara123) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Cloud_Discovery_Alert.yml]:
```suggestion
id: Prisma Cloud Compute - Cloud Discovery Alert
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Compliance_Alert.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+id: fcf94956-24cc-4515-8736-0b56a8c72f98
```
Comment (anara123) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Compliance_Alert.yml]:
```suggestion
id: Prisma Cloud Compute - Compliance Alert
```

File: Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Vulnerability_Alert.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
+id: 532d4a56-3e1a-44c1-8c8d-36cc117abd8f
```
Comment (anara123) [inline Packs/PrismaCloudCompute/Playbooks/Prisma_Cloud_Compute_-_Vulnerability_Alert.yml]:
```suggestion
id: Prisma Cloud Compute - Vulnerability Alert
```

**Comment by kostakhv** [comment]
All fixed @anara123 , thanks!

**Comment by anara123** [comment]
@kirbles19 please review - there are only 3 files to review -
PaloAltoNetworks_PrismaCloudCompute.yml
PaloAltoNetworks_PrismaCloudCompute_description.md
PaloAltoNetworks_PrismaCloudCompute/README.md

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**